### PR TITLE
[AC-8695] Remove role-based redirects

### PR DESCRIPTION
**Pull request by geofrocker**
https://masschallenge.atlassian.net/browse/AC-8695

### CODE REVIEW

File: accelerator/migrations/0044_remove_role_based_redirects.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                    1986, 2073, 2028, 2226, 1939, 1941, 2139, 2238, 2062, 1961,
+                    2060, 1985, 2072, 2027, 2225, 1938, 1942, 2138, 2237, 2063,
+                    2010, 1998, 1999, 1962, 1973, 1974, 1997, 2000, 2079, 2076,
+                    2074, 2235, 2078, 2077, 2075, 2234, 2162, 2161, 884, 43]
+
+
+def update_program_roles(apps, schema_editor):
+    ProgramRole = apps.get_model('accelerator', 'ProgramRole')
+    program_roles = ProgramRole.objects.exclude(id__in=PROGRAM_ROLE_IDS)
+    for program_role in program_roles:
```
Comment (jonkiparsky) [inline accelerator/migrations/0044_remove_role_based_redirects.py:16]: You can do this more efficiently with the `update` method:

`ProgramRole.objects.exclude(id__in=PROGRAM_ROLE_IDS).update(landing_page="")`

**Review by iankigen — COMMENTED** [review]
@geofrocker We have conflicting migrations

**Review by iankigen — APPROVED** [review]
Code/QA looks good
- Delete the data in the URL redirect field for all of the program roles in the attached .xlsx ✅

**Review by kenware — APPROVED** [review]
Looks good

**Pull request merged**